src/data-tutorials/users.tsx

```diff
@@ -1194,6 +1194,13 @@ const Users: User[] = [
     website: '/blog/2022/02/21/introduction-to-celo-progressive-dappstarter',
     tags: ['beginner','foundation', 'react', 'materialui', 'dappstarter', 'progressive'],
   },
+  {
+    title: 'Deploy and Interact with Smart Contracts on the Celo Blockchain',
+    description: 'Learn to deploy and interact with self-executing tamper-proof agreements developing smart contracts with Solidity on Celo Blockchain',
+    preview: require('./showcase/beginner/Deploy-and-Interact-with-Smart-Contracts-on-the-Celo-Blockchain.png'),
```

Comment:
Hey Fix you Image URL : showcase/intermediate/Deploy-and-Interact-with-Smart-Contracts-on-the-Celo-Blockchain.png